docs/bigquery-usage.rst

```diff
@@ -1,9 +1,5 @@
-.. toctree::
-  :maxdepth: 0
-  :hidden:
```

Comment:
@tseaver I'm not 100% sure about removing these. What is the point of the `toctree` in docs other than `index.rst`?